library/common/extensions/filters/http/platform_bridge/filter.h

```diff
@@ -50,6 +50,18 @@ class PlatformBridgeFilter final : public Http::PassThroughFilter,
 public:
   PlatformBridgeFilter(PlatformBridgeFilterConfigSharedPtr config);
 
+  // Scheduled on the dispatcher when resumeRequest is called from platform
+  // filter callbacks. Provides a snapshot of pending request state to the
+  // platform filter, and consumes invocation results to modify pending HTTP
+  // entities before resuming decoding.
+  void onResumeDecoding();
```

Comment:
I know decoding/encoding is an Envoy construct, but I think `onResumeRequest`/`onResumeResponse` would be clearer here since these are executed as a result of functions with similar names in Envoy Mobile

Reply:
The other functions in this class all use `decode` and `encode` (see L69+ below), due to the names being dictated by Envoy's own interfaces. I use request/response everywhere above this, because I do think it's clearer, especially for our use case, but my feeling was that it was worthwhile being consistent at least within a single class.